terra-jupyter-base/Dockerfile

```diff
@@ -98,7 +98,7 @@ RUN useradd -m -s /bin/bash -N -u $WELDER_UID $WELDER_USER
 ENV JUPYTER_PORT 8000
 ENV JUPYTER_HOME /etc/jupyter
 
-ENV PYTHONPATH $PYTHONPATH:$JUPYTER_HOME/custom
+ENV PYTHONPATH ${PYTHONPATH:+$PYTHONPATH:}$JUPYTER_HOME/custom
```

Comment:
Hm what does this syntax do?

Reply:
This took me a while to follow. Can we add a comment above it?